.github/workflows/image.yaml

```diff
@@ -0,0 +1,81 @@
+name: release
+
+on: 
+  create:
+  push:
+    branches:
+      - master
+      - 'v*'
+    tags:
+      - 'v*'
+
+# TODO: Migrate building binaries from Cloud Build
+jobs:
+  hnc-manager:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Set Release Tag
+        run: |
+          TAG="${GITHUB_REF#refs/*/v}"
+          echo "RELEASE_TAG=${TAG}" >> $GITHUB_ENV
+
+      - name: Set Build Date
+        id: date
+        run: |
+          echo "::set-output name=HNC_BUILD_TIMESTAMP::$(date -u +'%Y-%m-%dT%H:%M:%S%Z')"
+
+      - name: Set up Cloud SDK
+        uses: google-github-actions/setup-gcloud@master
+        with:
+        # GCR_KEY is the service account key in json which is base64 encoded and stored in the secrets section of the repo.
+          service_account_key: ${{ secrets.GCR_KEY }}
```

Comment:
Can you say anything about what this service account needs to be able to do? E.g., it must have permission to push to `gcr.io/k8s-staging-multitenancy`? If that's the requirement, how did you test this - by pushing to some different GCR repo?

Reply:
I used the `Default compute service account ` as shown in this [blog](http://acaird.github.io/computers/2020/02/11/github-google-container-cloud-run)  and yes, I tested with a different repo.

Reply:
Ah. That's bad advice, the default compute service account is terrible and generally shouldn't be used :D But there was no way for you to know that! I take it that the account was in the same GCP project as the repo you used to test?

Maybe just add the comment that we _think_ the requirement for the service accounts it that it can push to the required registry (sorry, "registry" not "repo") but that we're not certain. We'll find out soon enough!